ThunderCloud/Quiz.swift

```diff
@@ -108,8 +119,14 @@ open class QuizPage: StormObjectProtocol {
             if QuizConfiguration.shared.shuffleQuestions {
                 questions.shuffle()
             }
-            
-            self.questions = questions
+
+            if QuizConfiguration.shared.hasMaxNumberOfQuestions {
```

Comment:
```swift
if let maxNumberOfQuestions = maxNumberOfQuestions
```
it saves the need to have a default. Might even be able to drop `hasMaxNumberOfQuestions` entirely but I'm OK to leave that if you want

Reply:
Once this is actioned we can merge